### ISSUE TITLE
3.11.1. Static and 3.11.2 Shared chapters are missing

### ISSUE THREAD

**Issue by Roy-Kid** [ruslo/CGold#62]
https://cgold.readthedocs.io/en/latest/tutorials/libraries/static.html
https://cgold.readthedocs.io/en/latest/tutorials/libraries/shared.html

**Comment by ruslo** [comment]
Yes, the project is not under active development. I have added a note.
